### DISCUSSION TITLE
Guild

### DISCUSSION THREAD

**Discussion by Mina191** [Crownicles/Crownicles#1971] [Nouvelles idées]
I like the idea of guilds and having a guildshelter; although, I think it would be neat to have the possibility to upgrade both more: for example, once a guild reach a certain level - 10/20/30 etc -, to increase the number of members to invite and to have more animals to include in the shelter.

**Comment by ghost** [comment]
Hello, you couldn't know before but a similar suggestion already exists in french even if it isn't completely the same. Refer to #1965 and #598 by using a translator if you don't understand french.

**Reply by Mina191** [reply]
I will look for those now, thanks for the heads up.

**Reply by BastLast** [reply]
Don't hesitate to contact us through discord for any struggle relative to french content on GitHub :)
